packages/rn-tester-e2e/apps/put-app-here.txt

```diff
@@ -0,0 +1 @@
+put app in this folder
```

Comment:
nit: we should change the name of this file to README.txt so that it shows up by default if you navigate to it via ex. GH web ui